src/legacy/core_plugins/elasticsearch/lib/cluster.ts

```diff
@@ -0,0 +1,50 @@
+/*
+ * Licensed to Elasticsearch B.V. under one or more contributor
+ * license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright
+ * ownership. Elasticsearch B.V. licenses this file to you under
+ * the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import { errors } from 'elasticsearch';
+import { CallAPIOptions, ClusterClient } from 'kibana';
+
+export class Cluster {
```

Comment:
the file is in legacy folder because we are going to refactor it?

Reply:
We should remove `src/legacy/core_plugins/elasticsearch` entirely as soon as we can and use `server.core.elalasticsearch` (or whatever you decide in https://github.com/elastic/kibana/projects/13#card-17899106) instead.